lib/smart_app_launch/client_suite/access_alca_interaction_test.rb

```diff
@@ -14,6 +14,14 @@ class SMARTClientAccessAppLaunchConfidentialAsymmetricInteraction < Inferno::Tes
       using a SMART token obtained using the SMART App Launch EHR launch
       or standalone launch flow.
     )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@11',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@22',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@64',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@65',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@229',
```

Comment:
@229 should be an attestation. please remove.

@229
Use of the client credentials grant type requires that the client SHALL be a “confidential” client capable of protecting its authentication credential.

Reply:
Removed in [9b4d4ad](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/9b4d4adf0a9945b994d01c6ee21808ba05be91a6).